app/config/collections.php

```diff
@@ -1870,6 +1870,17 @@
                 'array' => false,
                 'filters' => [],
             ],
+            [
+                '$id' => ID::custom('name'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => ['json'],
```

Comment:
We won't the filter here once we use a plain string

Reply:
removed filter

Reply:
Seems it's still here 😅

Reply:
Ah apologies, file was not added to staging area